### DISCUSSION TITLE
Add ability to query Project Property across all projects

### DISCUSSION THREAD

**Discussion by 37b** [DependencyTrack/dependency-track#990] [Ideas]
I have been using tags to label projects with our in-house unique identifier for all components. I have written some jenkins shared libraries that search for projects using the `/v1​/project​/tag​/{tag}` API and if one does not exist it will create the project in DTrack and tag it.

I'd like to switch to using project properties. However, as far as I can tell there is no property equivalent for the tag search API

If would be great to see `/v1/project/property/?group={group}&name={name}&value={value}` or similar API as the project Search API is a little too noisy for my intended purpose.

**Comment by stevespringett** [comment]
I like this idea.  Although a POST containing a JSON object consisting of group, name, and value may be a better alternative as it would eliminate URL escaping requirements.
